recipes/pcapplusplus/all/test_package/conanfile.py

```diff
@@ -1,18 +1,40 @@
-from conans import ConanFile, CMake, tools
+from conan import ConanFile
+from conan.tools.build import can_run
+from conans import CMake
 import os
+import shutil
 
 
 class PcapplusplusTestConan(ConanFile):
     settings = "os", "compiler", "build_type", "arch"
     generators = "cmake"
+    # Needed to make sure libpcap gets compiled
+    build_policy = "missing"
```

Comment:
which profile is missing?

Reply:
shared lib, conan center does not provide shared libs

Reply:
Conan Center provides shared library. If it's missing, is because that package is incorrect or its package ID does not allow shared library.

Reply:
```suggestion
```